crates/nu-command/src/filters/each_while.rs

```diff
@@ -242,12 +263,15 @@ impl Command for EachWhile {
 
 #[cfg(test)]
 mod test {
-    use super::*;
+    use nu_test_support::{nu, pipeline};
 
     #[test]
-    fn test_examples() {
-        use crate::test_examples;
+    fn uses_optional_index_argument() {
+        let actual = nu!(
+            cwd: ".", pipeline(
+            r#"[7 8 9 10] | each while {|e i| $e + $i } | to nuon"#
+        ));
 
-        test_examples(EachWhile {})
```

Comment:
Did you mean to remove testing the examples?

Reply:
Ah, whoops, that explains that. 🦧